frontend/src/components/BenchmarkResults/BenchmarkTypeResultSection.tsx

```diff
@@ -0,0 +1,40 @@
+import Accordion from "react-bootstrap/Accordion";
+import { BenchmarkGroupResult } from "../../data/parseBenchmarkData";
+import ListGroup from "react-bootstrap/ListGroup";
+import BenchmarkResult from "./BenchmarkResult";
+import Badge from "react-bootstrap/Badge";
+
+const BenchmarkTypeResultSection = ({
+  benchmarkType,
+  benchmarkResults,
+}: {
+  benchmarkType: string;
+  benchmarkResults: BenchmarkGroupResult[];
+}) => {
+  return (
+    <ListGroup.Item
+      as="li"
+      className="d-flex justify-content-between align-items-start"
+    >
+      <div className="ms-2 me-auto">
+        <div className="fw-bold mb-4 mt-2 ">
+          Benchmark Type:{" "}
+          {benchmarkType.charAt(0).toUpperCase() + benchmarkType.slice(1)}
```

Comment:
I think we do this in a few places, is there some helper lib we can use with a title function?

Reply:
Will look for it